tilequeue/command.py

```diff
@@ -2468,6 +2468,27 @@ def command_fn(cfg, args):
                            help='Tile coordinate as "z/x/y".')
     subparser.add_argument('--run_id', required=False,
                            help='optional run_id used for logging')
+    subparser.add_argument('--postgresql_host', required=False,
+                           help='optional string of a list of db hosts e.g. '
+                                '`["aws.rds.url", "localhost"]`')
+    subparser.add_argument('--postgresql_dbnames', required=False,
+                           help='optional string of a list of db names e.g. '
+                                '`["gis"]`')
+    subparser.add_argument('--postgresql_user', required=False,
+                           help='optional string of db user e.g. `gisuser`')
+    subparser.add_argument('--postgresql_password', required=False,
+                           help='optional string of db password e.g. '
+                                '`VHcDuAS0SYx2tlgTvtbuCXwlvO4pAtiGCuScJFjq7wersdfqwer`')
+    subparser.add_argument('--store_name', required=False,
+                           help='optional string of a list of tile store '
+                                'names e.g. `["my-meta-tiles-us-east-1"]`')
+    subparser.add_argument('--store_date_prefix', required=False,
```

Comment:
I think we've been trying to change "date_prefix" to "build_id" elsewhere because they don't have to be date-related.

Reply:
> I think we've been trying to change "date_prefix" to "build_id" elsewhere because they don't have to be date-related.

we can probably change that to "build_id" in a separate PR since the renaming may require more changes.